src/modules/categories/components/Categories.js

```diff
@@ -152,30 +154,33 @@ class Categories extends React.Component<PropsType> {
     } else if (category.isRoot()) {
       // first level, we want to display a table with all first order categories
 
-      return <Tiles tiles={this.getTileModels(children)}
-                    title={CityModel.findCityName(cities, category.title)}
-                    onTilePress={this.onTilePress}
-                    navigateToFeedback={this.navigateToFeedbackOfCategories}
-                    theme={theme}
-                    t={t} />
+      return <SpaceBetween>
+        <View><Tiles tiles={this.getTileModels(children)}
```

Comment:
Using newlines makes this much clearer imo, same for `</View>`

Reply:
That's just taste I guess :P

Reply:
Really? You like it like that? :astonished: :astonished: But thanks ^^